src/components/Chart/Lines.js

```diff
@@ -0,0 +1,513 @@
+                  xFormat={xFormat}
+                  y={y}
+                  yTicks={yAxis.ticks}
+                  yAxisFormat={yAxis.axisFormat}
+                  confidence={confidence}
+                  yCut={yCut}
+                  yCutHeight={yCutHeight}
+                  yAnnotations={yAnnotations}
+                  endDy={endDy}
+                  width={innerWidth} />
+              </g>
+            )
+          })
+        }
+      </svg>
+      <div>
+        <div style={{paddingLeft, paddingRight}}>
+          <ColorLegend inline values={(
+            []
+              .concat(props.colorLegend && colorLegend && colorLegendValues)
+              .concat(!mini && confidence && {label: (
+                <span {...styles.confidenceLegend}>
+                  <span {...styles.confidenceBar} />
+                  {` ${t(`styleguide/charts/confidence${confidence}-legend`)}`}
+                </span>
+              )})
+              .filter(Boolean)
+          )}/>
+        </div>
+        {children}
+      </div>
+    </div>
+  )
+}
+
+LineChart.propTypes = {
+  children: PropTypes.node,
+  values: PropTypes.array.isRequired,
+  width: PropTypes.number.isRequired,
+  mini: PropTypes.bool,
+  x: PropTypes.string.isRequired,
+  xScale: PropTypes.oneOf(['time', 'ordinal']),
+  xSort: sortPropType,
+  xTicks: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+  timeParse: PropTypes.string.isRequired,
+  timeFormat: PropTypes.string.isRequired,
+  column: PropTypes.string,
+  columnSort: sortPropType,
+  columnFilter: PropTypes.arrayOf(PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    test: PropTypes.string.isRequired
+  })),
+  highlight: PropTypes.string,
+  stroke: PropTypes.string,
+  labelFilter: PropTypes.string,
+  color: PropTypes.string,
+  colorRange: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
+  colorRanges: PropTypes.shape({
+    sequential3: PropTypes.array.isRequired,
+    discrete: PropTypes.array.isRequired
+  }).isRequired,
+  colorLegend: PropTypes.bool,
+  category: PropTypes.string,
+  confidence: PropTypes.oneOf([95]),
+  numberFormat: PropTypes.string.isRequired,
+  zero: PropTypes.bool.isRequired,
+  filter: PropTypes.string,
+  startValue: PropTypes.bool.isRequired,
+  endLabel: PropTypes.bool.isRequired,
+  endDy: PropTypes.string.isRequired,
+  minInnerWidth: PropTypes.number.isRequired,
+  columns: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+  unit: PropTypes.string,
+  yAnnotations: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.number.isRequired,
+    label: PropTypes.string.isRequired,
+    x: PropTypes.string,
+    dy: PropTypes.string
+  })),
+  t: PropTypes.func.isRequired,
+  description: PropTypes.string
+}
+
+export const Line = props => <LineChart {...props} />
+
+Line.defaultProps = {
+  x: 'year',
+  xScale: 'time',
+  timeParse: '%Y',
+  timeFormat: '%Y',
+  numberFormat: '.0%',
+  zero: true,
```

Comment:
Waaaat? ;)
Looks like this triggers a shortened/compact axis. At some point we should find a more meaningful name for this prop. Just saying, OK for now.